ui/static/ui/js/learning_resources.jsx

```diff
@@ -369,16 +368,16 @@ define('learning_resources', [
           return;
         }
 
-        var contentXml = data.content_xml;
         var learningResourceType = data.learning_resource_type;
         var description = data.description;
+        var title = data.title;
         var selectedTerms = data.terms;
         var previewUrl = data.preview_url;
 
         thiz.setState({
-          contentXml: contentXml,
           message: undefined,
           description: description,
+          title: title,
```

Comment:
I'd say probably since we don't need it in this PR, we can just add it in the next one, but this is so small it doesn't matter either way